src/Bundler.js

```diff
@@ -135,7 +135,8 @@ class Bundler extends EventEmitter {
       detailedReport: options.detailedReport || false,
       global: options.global,
       autoinstall:
-        typeof options.autoinstall === 'boolean'
+        typeof options.autoinstall === 'boolean' &&
+        process.env.PARCEL_NO_AUTOINSTALL !== 'true'
```

Comment:
I think it would be better if the options were positive instead of negative, so `PARCEL_AUTOINSTALL=false` should disable it

Reply:
That was my original thought. Brb updating.